LoopKitUI/Views/PseudoNavigationBar.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  PseudoNavigationBar.swift
+//  LoopKitUI
+//
+//  Created by Michael Pangburn on 5/7/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+
+struct PseudoNavigationBar<Leading: View, Center: View, Trailing: View>: View {
```

Comment:
Was there a problem in actually using a NavigationView to provide this? Just worried about future iOS design changes making a custom impl turn into bitrot.

Reply:
I'm not happy with this either. NavigationView doesn't to work well in miniaturized form—bar button items become untappable when the frame's height is constrained. 

The mini-modal design doesn't have to be UINavigationBar-esque, if you have other ideas; originally I had a large rounded 'Add' button at the bottom, though this felt more intuitive.

![IMG_9FBA040919C9-1](https://user-images.githubusercontent.com/25555889/81723119-3ade9800-9437-11ea-9d0b-05f0354d830d.jpeg)